app/api/actions/github/route.ts

```diff
@@ -324,174 +323,210 @@ export async function POST(request: Request) {
-        value: confluence,
-        userLogin,
-        systemName: "Confluence",
-        systemResponseName: "Confluence Docs",
-      });
-      textToWrite += generalMarkdownHelper({
-        value: slack,
-        userLogin,
-        systemName: "Slack",
-        systemResponseName: "Slack Threads",
-      });
-      textToWrite += generalMarkdownHelper({
-        value: notion,
-        userLogin,
-        systemName: "Notion",
-        systemResponseName: "Notion Pages",
-      });
-      textToWrite += generalMarkdownHelper({
-        value: linear,
-        userLogin,
-        systemName: "Linear",
-        systemResponseName: "Linear Tickets",
-      });
-      textToWrite += generalMarkdownHelper({
-        value: asana,
-        userLogin,
-        systemName: "Asana",
-        systemResponseName: "Asana Tasks",
-      });
-      textToWrite += countMarkdown({
-        count,
-        isPrivateRepo: repository.private,
-        repoName: repo,
-      });
+        textToWrite += generalMarkdownHelper({
+          value: github,
+          userLogin,
+          systemName: "GitHub",
+          systemResponseName: "GitHub PRs",
+        });
+        textToWrite += generalMarkdownHelper({
+          value: jira,
+          userLogin,
+          systemName: "Jira",
+          systemResponseName: "Jira Tickets",
+        });
+        textToWrite += generalMarkdownHelper({
+          value: confluence,
+          userLogin,
+          systemName: "Confluence",
+          systemResponseName: "Confluence Docs",
+        });
+        textToWrite += generalMarkdownHelper({
+          value: slack,
+          userLogin,
+          systemName: "Slack",
+          systemResponseName: "Slack Threads",
+        });
+        textToWrite += generalMarkdownHelper({
+          value: notion,
+          userLogin,
+          systemName: "Notion",
+          systemResponseName: "Notion Pages",
+        });
+        textToWrite += generalMarkdownHelper({
+          value: linear,
+          userLogin,
+          systemName: "Linear",
+          systemResponseName: "Linear Tickets",
+        });
+        textToWrite += generalMarkdownHelper({
+          value: asana,
+          userLogin,
+          systemName: "Asana",
+          systemResponseName: "Asana Tasks",
+        });
+        textToWrite += countMarkdown({
+          count,
+          isPrivateRepo: repository.private,
+          repoName: repo,
+        });
 
-      // Make Watermelon Review the PR's business logic here by comparing the title with the AI-generated summary
-      await labelPullRequest({
-        prTitle: title,
-        businessLogicSummary,
-        repo,
-        owner,
-        issue_number: number,
-        installationId,
-        reqUrl: request.url,
-        reqEmail: req.email
-      });
+        // Make Watermelon Review the PR's business logic here by comparing the title with the AI-generated summary
+        await labelPullRequest({
```

Comment:
Do we want the label anyway?

Reply:
Sorry I don't quite understand this comment. This line, 408, is inside the if statement that runs the logic for being below the query threshold.